packages/main/src/i18n/messagebundle.properties

```diff
@@ -229,6 +229,15 @@ SWITCH_ON=On
 #XACT: ARIA announcement for the switch off
 SWITCH_OFF=Off
 
+#XACT: ACC next icon name in tab container 
+TABCONTAINER_NEXT_ICON_ACC_NAME=Next
+
+#XACT: ACC previous icon name in tab container
+TABCONTAINER_PREVIOUS_ACC_NAME=Previous
```

Comment:
Just this small thing, let's be consistent, 
we have 
TABCONTAINER_NEXT_ICON_ACC_NAME, but TABCONTAINER_PREVIOUS_ACC_NAME (without icon in the name).

Rename it to TABCONTAINER_PREVIOUS_ICON_ACC_NAME and don`t forget to update the place you import them.

Reply:
Done